plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryTypeMapping.java

```diff
@@ -335,6 +335,81 @@ public Object[][] bigqueryUnsupportedBigNumericTypeProvider()
         };
     }
 
+    @Test(dataProvider = "bigqueryNumericTypeProvider")
+    public void testNumericWriteMapping(String type, String value)
```

Comment:
nit: Move to after `testNumericMapping`.

Reply:
Why can't we use `SqlDataTypeTest`? It worth to leave a comment. Otherwise, someone may try to migrate the test.

Reply:
Moved and comment added